css/index.css

```diff
@@ -65,6 +65,50 @@ h1, h2, h3, h4, h5 {
 p {
```

Comment:
When running your page in live server, the header section is a lot bigger than the rest of the page.  You can make this the same size as the rest of the content on your page by setting the width of the header to the same size as the container class.